model_analyzer/triton/server/server_config.py

```diff
@@ -123,6 +123,31 @@ def to_cli_string(self):
         return ' '.join(
             [f'--{key}={val}' for key, val in self._server_args.items() if val])
 
+    def to_args_list(self):
+        """
+        Utility function to convert a cli string into a list of arguments while
+        taking into account "smart" delimiters.  Notice in the example below
+        that only the first equals sign is used as split delimiter.
+
+        Returns
+        -------
+        list
+            the list of arguments consisting of all set arguments to
+            the tritonserver.
+
+            Example:
+            input cli_string: "--model-control-mode=explicit 
+                --backend-config=tensorflow,version=2"
+
+            output: ['--model-control-mode', 'explicit', 
+                '--backend-config', 'tensorflow,version=2']
+        """
+        list = []
```

Comment:
Could you please only rename this `list` variable to `args_list`? `list` is already defined and might create weird issues. No need to rerun the CI after the variable change. Thank you.

Reply:
Done.